src/runtime/pickle_runner.ts

```diff
@@ -243,6 +277,11 @@ export default class PickleRunner {
       this.result.status === Status.FAILED &&
       attempt + 1 < this.maxAttempts
     ) {
+      /*
+      TODO dont rely on `testStepResult.willBeRetried`, it will be moved or removed
+      see https://github.com/cucumber/cucumber/issues/902
+       */
```

Comment:
Can we make a cucumber-js issue for this?

Reply:
Yep raised #1409